data/pizza.json

```diff
@@ -747,8 +747,8 @@
 				"hidden_ingredients": [
 					"tomatsås", "ost"
 				],
-				"price": 100,
-				"family_price": 220
+				"price": 105,
+				"family_price": 229
 			},
 			"daniel special": {
```

Comment:
[Fårost instead of fetaost](https://sannegarden.com/meny/johanneberg/#:~:text=Champinjoner%2C%20Paprika%2C%20Vitl%C3%B6kss%C3%A5s%2C%20F%C3%A5rost%2C%20Gorgonzolaost)